src/APIM_ARMTemplate/apimtemplate/Common/Templates/Policy/PolicyTemplateUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿using apimtemplate.Extractor.Models;
+using Microsoft.Extensions.Caching.Memory;
+using System.IO;
+
+namespace apimtemplate.Common.Templates.Policy
+{
+    internal static class PolicyTemplateUtils
+    {
+        private static readonly IMemoryCache _policyCache = new MemoryCache(new MemoryCacheOptions());
+
+        public static string GetPolicyContent(ExtractorParameters extractorParameters, PolicyTemplateResource policyTemplateResource)
+        {
```

Comment:
What about nulls?

Reply:
again just moved the code, didn't refactor it right now. Only changing the structure in this PR